src/qmozenginesettings.h

```diff
@@ -27,6 +27,7 @@ class QMozEngineSettings : public QObject {
     Q_PROPERTY(bool useDownloadDir READ useDownloadDir WRITE setUseDownloadDir NOTIFY useDownloadDirChanged)
     Q_PROPERTY(QString downloadDir READ downloadDir WRITE setDownloadDir NOTIFY downloadDirChanged)
     Q_PROPERTY(qreal pixelRatio READ pixelRatio WRITE setPixelRatio NOTIFY pixelRatioChanged)
+    Q_PROPERTY(bool doNotTrackEnabled READ doNotTrackEnabled WRITE setDoNotTrackEnabled NOTIFY doNotTrackEnabledChanged)
```

Comment:
```suggestion
    Q_PROPERTY(bool doNotTrack READ doNotTrack WRITE setDoNotTrack NOTIFY doNotTrackChanged)
```
=> doNotTrack == true already state that it is enabled, doesn't it.